md/bdm-management-in-bonita-bpm-portal.md

```diff
@@ -41,25 +53,27 @@ In such cases, you will have to implement the change on your own, through carefu
 This limitation is well known and will be addressed in a future Bonita version.
 :::
 
-::: info
-**Note:** For Enterprise, Efficiency and Performance editions only.
-:::
-
 <a id="installAccessControl"/>
+## Manage BDM Access Control
 
-It is possible to define Business Data Model access control rules in Bonita Studio and import them in Bonita Portal.  
-The Access control rules must match the Business Data model.
-
-1. To import the BDM access control rules created in Bonita Studio, first [export the Access Control file](bdm-access-control.md) from Bonita Studio where it was designed.
-2. In the Bonita Portal, log in as the technical user (default login 'install', default password 'install') or as a user with the administrator profile.
-3. As the tenant technical user or as an administrator, go to the **BDM** menu.
-4. A page with a section **Business Data Model access control** is displayed.
-5. Click **_Install_** to open import popup and choose the file containing the Business Data Model access control definition exported from Bonita Studio, and click on _**Install**_.
-6. A successful import message will be displayed.
+::: info
+**Note:** For Enterprise, Performance, and Efficiency editions only.
+:::
 
-Please note that a delay is required after the import for the engine to process the access control and for the filtering to be effective.
+It is possible to define BDM Access Control rules in Bonita Studio and import them in Bonita Portal.  
+The Access Control rules must match the Business Data model.  
 
 ::: info
-**:fa-info-circle: Note:** Contrary to the BDM definition, the deployment of a Business Data Model access control file doesn't require pausing the BPM services during the operation. 
+**:fa-info-circle: Note:** Contrary to the BDM definition, the deployment of a BDM Access Control file does not require pausing the BPM services during the operation. 
 :::
+1. [Export the BDM Access Control file](bdm-access-control.md) from Bonita Studio where it was designed.
+1. In Bonita Portal, log in as a user with the _Administrator_ profile, or as the Technical user (default login 'install', default password 'install') 
```

Comment:
I would remove the default username/password as it is necessarily changed by Sam

Reply:
ok